python/fastquant/fastquant.py

```diff
@@ -567,3 +572,69 @@ def datestring_to_datetime(date, sep="-"):
     errmsg = "date format must be YYYY-MM-DD"
     assert len(ymd[0]) == 4, errmsg
     return datetime(*map(int, ymd))
+
+
+def get_bt_news(keyword=None, page_nums=None):
+    """
+    This function scrapes Business Times (https://www.businesstimes.com.sg) articles by giving
+    a specific keyword e.g "facebook, jollibee" and number of pages that you needed.
+
+    Parameters
+    ----------
+    keyword : str
+        The keyword you wanted to search for in Business Times page.
+    page_nums : int
+        The number of iteration of pages you want to scrape.
+
+    Returns
+    ----------
+    date_sentiments: dict
+        The dictionary output of the scraped data in form of {date: sentiment score}
+    """
+
+    nltk.download("vader_lexicon", quiet=True)  # download vader lexicon
+
+    # Check for keyword
+    if keyword is None:
+        print(
+            "keyword is set to None, please type in keyword or company name that you wanted to scrape e.g jollibee, facebook etc."
+        )
+        raise TypeError
+    if page_nums is None:
+        page_nums = 1
+        print("no page numbers indicated, setting this variable to 1")
+
+    date_sentiments = {}
+    sia = SentimentIntensityAnalyzer()
+
+    for i in tqdm(range(1, page_nums + 1)):
+        page = urlopen(
+            f"https://www.businesstimes.com.sg/search/{keyword}?page={str(i)}"
```

Comment:
```suggestion
            "https://www.businesstimes.com.sg/search/{}?page={}".format(keyword, str(i))
```
f-string is not yet supported for python 3.4 users

Reply:
For context @rafmacalaba , we agreed previously to stick to the old format method for now to leave space for earlier python 3 versions :)